PCP_Chapter4_Exercises.cpp

```diff
@@ -14,51 +14,69 @@
 // Questions:
 //  How can I call a class that is in a different source file to the main function? 
 
-#include "stdafx.h"
+// use "using namespace blabla" sparingly - in general it is better to type out
+// the whole namespace to avoid ambiguity, and make the source clear.
+
+#include <iostream>
 using namespace std;
 
-// Called ex_42 for the exercise problem numbered 4.2 in the book. Holds functions that return a letter block (So far Hello) 
-class ex_42 {
```

Comment:
Classes by convention must start with a capital letter, and better not contain special characters.